### PR TITLE
feat: added an Anime Template Portfolio

### PR DESCRIPTION
**Pull request by coder-ralph**

### CODE REVIEW

File: anime/assets/script.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+let menu = document.querySelector('#menu-icon');
+let navbar = document.querySelector('.navbar');
+
+menu.onclick = () => {
+    menu.classList.toggle('bx-x');
```
Comment (jofftiquez) [inline anime/assets/script.js:5]: As much as possible we are using 2 space tabs for our indentation. Kindly revise, thanks

File: anime/assets/styles.css [hunk 2]
```diff
@@ -0,0 +1,431 @@
+*{
+    padding: 0;
```
Comment (jofftiquez) [inline anime/assets/styles.css:2]: 2 spaces for css as well

File: anime/index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (jofftiquez) [inline anime/index.html:5]: 2 spaces for html as well

File: anime/index.html
Comment (jofftiquez) [inline anime/index.html:1]: Please update the root index.html and add your theme there. 

Reference: https://github.com/OSSPhilippines/freefolio/blob/main/index.html

**Review by jofftiquez — CHANGES_REQUESTED** [review]
So far looking good, kindly resolve the few comments, thanks

File: anime/assets/script.js (hunk 1 above)
Comment (coder-ralph) [reply anime/assets/script.js:5]: Okay po Sir. Thank you so much for the feedback!

**Comment by jofftiquez** [comment]
Hello @coder-ralph has this been updated? Please mark the resolved comments as done if they are already addressed. Thank you <3

**Comment by coder-ralph** [comment]
I'm sorry, Sir, I forgot to click the "resolve conversation" button, but I want to inform you that all the comments have been resolved. The pull request is now ready for review/merge. Thank you Sir for your feedback, and please let me know if there's anything else I can improve.

**Comment by jofftiquez** [comment]
> I'm sorry, Sir, I forgot to click the "resolve conversation" button, but I want to inform you that all the comments have been resolved. The pull request is now ready for review/merge. Thank you Sir for your feedback, and please let me know if there's anything else I can improve.

Thank you, no problem @coder-ralph will review it as soon as I can thanks

**Comment by jofftiquez** [comment]
Closing and reopening the PR to re-run the Github workflow

**Comment by jofftiquez** [comment]
Reopening

**Review by jofftiquez — APPROVED** [review]
Looks great @coder-ralph approved. Thank you.

**Pull request merged**